.circleci/config.yml

```diff
@@ -12,15 +12,14 @@ commands:
       - checkout
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
-            - v1-dependencies-{{ checksum "package.json" }}
-            - v1-dependencies-
+            - v2-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
+            - v2-dependencies-{{ checksum "package.json" }}
+            - v2-dependencies-
       - run: yarn install
       - save_cache:
           paths:
-            - .git
```

Comment:
I don't know why I added it in my previous patch, probably a wrong copy/paste.
I even think it could bring a bad result because the checkout happens before :/